### ISSUE TITLE
Using telethon with flask: There is no current event loop in thread 'Thread-1'.

### ISSUE THREAD

**Issue by gekich** [LonamiWebs/Telethon#1208]
I want to use Telethon from the **Flask** Web App. But when I am running it, I am getting following error:

`RuntimeError: There is no current event loop in thread 'Thread-1'.`

**Code that causes the issue**
```python
from flask import Flask
from telethon.sync import TelegramClient


app = Flask(__name__)

@app.route('/')
def index():
    api_id = XXXXXX
    api_hash = 'XXXXXXX'
    bot_token = 'XXXXXXX'
    client = TelegramClient('bot', api_id, api_hash)
    client.start()
    return 'Index Page'

if __name__ == '__main__':
    app.run()
```

I tried to fix the issue using **sync-stale** version of Telethon. But there are no type **ChatBannedRights** i need.  
i used commands: 
`pip3 uninstall telethon`
`pip3 install --no-cache --upgrade git+https://github.com/LonamiWebs/Telethon@sync-stale`
to install the sync-stale version. 

And got error `ImportError: cannot import name 'ChatBannedRights' from 'telethon.tl.types' (/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/telethon/tl/types/__init__.py)` when tried to import type:

```python
from telethon.tl.types import ChatBannedRights
```

So basically there are two issues: 
1. Im getting error "There is no current event loop in thread 'Thread-1'." while using Flask and Telethon
2. There are no type ChatBannedRights in sync-stale version of Telethon. 

Hope you can help me

**Comment by Lonami** [comment]
Nope, sorry, **issues aren't for questions**. [StackOverflow](https://stackoverflow.com) or [the group](https://t.me/TelethonChat) are.

**Comment by ftuyama** [comment]
Why so rude? 
If someone has the same question: https://docs.telethon.dev/en/latest/concepts/asyncio.html?highlight=thread#can-i-use-threads

**Comment by Lonami** [comment]
You asked the same thing in #1253, there is no need to repeat yourself. Please stop reviving old issues.

**Comment by DaemonOld** [comment]
Hey. It works for me
```python
import asyncio

class TgClient(TelegramClient):

    def __init__(self, name):
        loop = asyncio.new_event_loop()
        asyncio.set_event_loop(loop)
        super().__init__(name, api_id, api_hash, loop=loop)

@app.route('/')
def index():
    client = TgClient('mysession')
    client.start()
    return 'Index Page'
```

**Comment by Thambolo** [comment]
> Hey. It works for me
> 
> ```python
> import asyncio
> 
> @app.route('/')
> def index():
>     loop = asyncio.new_event_loop()
>     asyncio.set_event_loop(loop)
>     client = TelegramClient('mysession', api_id, api_hash, loop=loop)
>     client.start()
>     return 'Index Page'
> ```

Hey there, does anyone know if you have to add `asyncio.set_event_loop(asyncio.new_event_loop())` to the start of every route or just at an initial route?

What if you want to do a client.run_until_disconnected() for events like newmessage, do you add it to every route or ? 

Thanks!

**Comment by FujiwaraChoki** [comment]
I still get the error, even after doing this

> Hey. It works for me
> 
> ```python
> import asyncio
> 
> @app.route('/')
> def index():
>     loop = asyncio.new_event_loop()
>     asyncio.set_event_loop(loop)
>     client = TelegramClient('mysession', api_id, api_hash, loop=loop)
>     client.start()
>     return 'Index Page'
> ```

**Comment by bovaqulov** [comment]
telethon and flask webhook sync
